devicetypes/smartthings/ikea-button.src/ikea-button.groovy

```diff
@@ -296,10 +296,6 @@ private Map getButtonEvent(Map descMap) {
 	}
 
 	if (buttonNumber != 0) {
-		// Create old style
-		def descriptionText = "${getButtonName(buttonNumber)} was $buttonState"
-		result = [name: "button", value: buttonState, data: [buttonNumber: buttonNumber], descriptionText: descriptionText, isStateChange: true]
```

Comment:
@tpmanley and @dkirker is this the right direction to go in for mcd now?

I'd favor keeping the "extra" event and simply marking the main event as `displayed: false`, but I can't really think of a compelling reason to keep it around. I guess doing it this way does make the main device on the device list change when you press any button, which seems good, right?

Reply:
I think we still need the event to ensure that the device works correctly in SmartApps like Smart Lighting. If the user selects the parent device followed by the button number it expects to be created on the parent device and will not see the events creates on the child components.

Reply:
@greens @tpmanley i reverted previous changes and added `displayed: false` to a parent event according to Your suggestions.